.github/workflows/end_to_end_test.yaml

```diff
@@ -12,6 +12,14 @@ jobs:
     steps:
       - name: Echo hello world
         run: echo "hello world"
+      - name: Proxy set in /etc/environment
+        run: cat /etc/environment | grep PROXY
+      - name: Install microk8s
+        run: sudo snap install microk8s --classic
+      - name: Wait for microk8s
+        run: sudo microk8s status --wait-ready
+      - name: Deploy nginx for testing
+        run: sudo microk8s kubectl create deployment nginx --image=nginx
```

Comment:
Should we also include a check that the deployment actually worked? Like a curl or something?